adsmp/tasks.py

```diff
@@ -35,9 +35,12 @@ def task_update_record(msg):
         - bibcode
         - and specific payload
     """
-    logger.debug('Updating record: %s', msg)
+    # logger.debug('Updating record: %s', msg)
+    logger.info('Updating record: %s', msg)
     status = app.get_msg_status(msg)
+    logger.info(f'Message status: {status}')
```

Comment:
These could probably become debug statements long term so we aren't flooding the logs.